datafusion/core/tests/dataframe/mod.rs

```diff
@@ -2400,7 +2400,8 @@ async fn write_json_with_order() -> Result<()> {
 #[tokio::test]
 async fn write_table_with_order() -> Result<()> {
     let tmp_dir = TempDir::new()?;
-    let ctx = SessionContext::new();
+    let config = SessionConfig::new().with_parquet_force_view_metadata(false);
```

Comment:
Why we need to disable this?

I can explain, because
1. We select from the datasource or memory which are not forced to utf8view type, but we create external table with utf8view field when we enable the force_view.  
2. When we insert into cases, we check the insert data type and the table data type, so we should make the data type consistent.

Related ticket to support cast it to utf8view automaticlly:
https://github.com/apache/datafusion/issues/13408#issue-2657679523
```rust
statement ok
create table t(a varchar) as values ('1'), ('2');

query T
select arrow_typeof(a) from t;
----
Utf8
Utf8

statement ok
drop table t
```